### PR TITLE
chore: add PR template

### PR DESCRIPTION
**Pull request by Splaktar**
Relates to #621

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<!-- 
+Filling out this template is required. Do not delete the template when submitting a Pull Request!
+-->
+## PR Checklist
```
Comment (jelbourn) [inline .github/PULL_REQUEST_TEMPLATE.md]: Personally I'm not a fan of this kind of cumbersome checklist. Could we simplify it to just 3 or four header questions?

Comment (Splaktar) [reply .github/PULL_REQUEST_TEMPLATE.md]: OK, I've simplified it a good deal. What do you think now?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Please check that your PR fulfills the following requirements:
+- [ ] The commit message follows [our guidelines](https://github.com/angular/components/blob/master/CONTRIBUTING.md#commit)
+- [ ] Tests for the changes have been added or weren't appropriate
+
+## What is the current behavior?
+<!-- Please describe the current behavior that you are modifying and
+     link to one or more relevant issues. -->
+
+<!-- Please do not open PRs without first opening an issue that describes the current
+     and expected behavior. -->
```
Comment (jelbourn) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would just remove these two comments

Comment (Splaktar) [reply .github/PULL_REQUEST_TEMPLATE.md]: Done.

**Pull request merged**